### PR TITLE
Add page for react-native-permissions component

### PR DESCRIPTION
**Pull request by tritao**
103020487-60409200-4540-11eb-902b-fdba8d41ab75

### CODE REVIEW

File: src/examples/PermissionsExamplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
+'use strict';
+import {Button, FlatList, Linking, Text, View} from 'react-native';
+import React, { useEffect, useState } from 'react';
```
Comment (chiaramooney) [inline src/examples/PermissionsExamplePage.tsx:3]: nit: remove extra spaces

File: src/examples/PermissionsExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+
```
Comment (chiaramooney) [inline src/examples/PermissionsExamplePage.tsx:6]: nit: remove extra new line

File: src/examples/PermissionsExamplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import {check, Permission, PERMISSIONS, PermissionStatus, request, RESULTS, WindowsPermission} from 'react-native-permissions';
+import { AndroidPermissionMap } from 'react-native-permissions/dist/typescript/permissions.android';
+import { IOSPermissionMap } from 'react-native-permissions/dist/typescript/permissions.ios';
+import { WindowsPermissionMap } from 'react-native-permissions/dist/typescript/permissions.windows';
```
Comment (chiaramooney) [inline src/examples/PermissionsExamplePage.tsx:10]: nit: remove extra spaces inside brackets

File: src/examples/PermissionsExamplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    );
+  }
+
+  const entries = Array.from(perms.entries());
+
+  return (
+    <Page
+      title="Permissions"
+      description="Allows requesting and showing available system permissions via the react-native-permissions module.">
+
```
Comment (chiaramooney) [inline src/examples/PermissionsExamplePage.tsx:104]: nit: remove extra new line

File: src/examples/PermissionsExamplePage.tsx [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+      <View>
+        <Text>For more information about the supported APIs, check out the{' '}
+          <Text style={{color: 'blue'}} onPress={() =>
+             Linking.openURL('https://github.com/zoontek/react-native-permissions#api')}>
+          documentation
+          </Text>
+        .
+        </Text>
+      </View>
```
Comment (chiaramooney) [inline src/examples/PermissionsExamplePage.tsx:113]: PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page in replacement of this?

**Comment by tritao** [comment]
Updated.

**Review by chiaramooney — APPROVED** [review]
👍 looks great

**Comment by tritao** [comment]
@chiaramooney Rebased and CI passing.

**Pull request merged**